bendowen/BEN_DOWEN.md

```diff
@@ -0,0 +1,74 @@
+# Tester of The Day
+
+## Origin story
+
+The story starts, as with all good things, with a joke. Dan Ashby had started a rant on Twitter about Paid-for awards, around the end of July 2020.
+Lot's of people joined in and thread and the jokes rumbled on for a few days. Many of us making more outranges sounding awards.
+
+You can checkout the thread for yourself: <https://twitter.com/DanAshby04/status/1288763581333413888>
+
+Inspired by the humour of this all, I thought, what if? What if I made a daily award, a free award?
+
+At first I thought I would simply find all the winners myself. Simply look on Twitter, find someone saying something cool about Software Testing and then boom, award them Tester of The Day.
+
+A few days in, and I was thinking maybe this will last a week. But then something amazing happened. The awesome power of the Software Testing Community awoke. People started to nominate people to win Tester of the Day.
```

Comment:
How about something like : ` I originally thought "this will last maybe a week". But then something amazing happened. People started to nominate others in the Software Testing Community as "Tester of the Day".`